webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -1,35 +1,51 @@
 import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import { Tabs, Tab, TabTitleText, Label } from '@patternfly/react-core';
 import { InfoCircleIcon } from '@patternfly/react-icons';
-
+import SkeletonLoader from 'foremanReact/components/common/SkeletonLoader';
 import { translate as __ } from 'foremanReact/common/I18n';
 
 import './AnsibleHostDetail.scss';
+import WrappedAnsibleHostInventory from './components/AnsibleHostInventory';
 
-const AnsibleHostDetail = props => {
+const AnsibleHostDetail = ({ response, status }) => {
   // https://projects.theforeman.org/issues/32398
-  const [activeTab] = useState('variables');
+  const [activeTab, setActiveTab] = useState('variables');
+  const handleTabClick = (event, tab) => setActiveTab(tab);
 
   return (
-    <Tabs activeKey={activeTab} isSecondary>
-      <Tab
-        eventKey="variables"
-        title={<TabTitleText>{__('Variables')}</TabTitleText>}
-      >
-        <div className="host-details-tab-item">
-          <div className="ansible-host-detail">
-            <Label
-              color="blue"
-              icon={<InfoCircleIcon />}
-              style={{ marginTop: '1.5rem' }}
-            >
-              Ansible Variables coming soon!
-            </Label>
+    <SkeletonLoader status={status} skeletonProps={{ count: 5 }}>
+      <Tabs onSelect={handleTabClick} activeKey={activeTab} isSecondary>
```

Comment:
If the Skeleton's children won't be undefined, an empty state won't be displayed, this fixes it:
```js
<SkeletonLoader>
  {response && (<Tabs>...
```